README.md

```diff
@@ -60,11 +59,11 @@ Check out our open source wall panel [here](https://github.com/micro-nova/AmpliP
 
 ## Speakers
 
-Notes on picking and installing speakers for whole house audio can be found [here](docs/SPEAKERS.md).
+Notes on picking and installing speakers for whole house audio can be found [here](https://amplipi.discourse.group/t/tested-recommended-speakers-and-speaker-wire/31).
```

Comment:
This SPEAKERS.md file still exists, but was literally just a link to the discourse post